src/components/landing/EventMatrix/EventMatrix.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import EventCard from "../EventCard/EventCard";
+import Image from "../../../mockup-images/football-g10e9bda93_1920.jpg";
+import "./EventMatrix.scss";
+
+export default function EventMatrix({ events }) {
+  return (
+    <div className="event-matrix">
+      <div className="event-matrix__body">
```

Comment:
Please remove this div and move styles to wrapper div

Reply:
Removed
DONE